src/sensai/featuregen.py

```diff
@@ -72,6 +72,11 @@ def __init__(self, categoricalFeatureNames: Optional[Union[Sequence[str], str]]
         self._name = None
         self.__isFitted = False
 
+    # for backwards compatibility with persisted Featuregens based on code prior to commit 7088cbbe
+    # They lack the __isFitted attribute and we assume that each such Featuregen was fitted
+    def __setstate__(self, d):
+        self.__isFitted = d.get("__isFitted", True)
```

Comment:
Also needs super call.

Reply:
see comment above with the snippet from torch_base. For some reason pycharm complains about super().__setstate__, I was put off by that

Reply:
`super().__setstate__` is the most robust variant, because setting `self.__dict__` (which is the default implementation in object) is not necessarily OK if we later introduce a super-class.